src/app/nextversion/nextversion_test.go

```diff
@@ -105,11 +115,11 @@ changes:
 			app.Writer = buf
 
 			err = app.Run(strings.Fields(fmt.Sprintf("rt -yaml %s %s next-version %s", yamlPath, tc.globalargs, tc.args)))
-			if err != nil {
-				t.Fatalf("Error running app: %v", err)
+			if !errors.Is(err, tc.errorExpected) {
+				t.Fatalf("Expected %v, got %v", tc.errorExpected, err)
 			}
 
-			if actual := buf.String(); actual != tc.expected+"\n" {
+			if actual := buf.String(); tc.expected != "" && actual != tc.expected+"\n" {
```

Comment:
Can we make this a different `if` earlier? Human brains suck reading nested conditions.

Reply:
done